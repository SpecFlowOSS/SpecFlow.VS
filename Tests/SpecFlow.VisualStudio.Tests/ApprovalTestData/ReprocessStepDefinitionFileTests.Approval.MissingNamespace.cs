[Binding]
public class Feature1StepDefinitions
{
    [When(@"I press add")]
    public void WhenIPressAdd()
    {
        throw new PendingStepException();
    }
}
