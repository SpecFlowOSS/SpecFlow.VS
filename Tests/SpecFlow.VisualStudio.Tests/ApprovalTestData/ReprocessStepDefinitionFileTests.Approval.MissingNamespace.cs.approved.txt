ProjectBindingRegistry:
  ProjectStepDefinitionBinding-0:
    IsValid:`True`
    Error:``
    StepDefinitionType:`When`
    SpecifiedExpression:`I press add`
    Regex:`^I press add$`
    Scope:``
    Implementation:
      Method:`Feature1StepDefinitions.WhenIPressAdd`
      ParameterTypes:[0]
      SourceLocation:
        SourceFile:`C:\<Full path to>\MissingNamespace.cs`
        SourceFileColumn:`5`
        SourceFileEndColumn:`5`
        SourceFileEndLine:`8`
        SourceLocationSpan:``
        SourceFileLine:`6`
    Expression:`I press add`
ProjectBindingRegistry:end
